src/content/en/2020/cms.md

```diff
@@ -23,3 +22,364 @@ published: 2020-12-09T00:00:00.000Z
+
+
+## What’s the user experience of CMS-powered websites?
+
+CMSs must offer a good user experience. With so much of the web relying on CMSs to serve pages, it’s the responsibility of the CMS at the platform-level to ensure the user experience is good. Our aim is to shed light on real-world user experience when using CMS-powered websites.
+
+To achieve this, we turn our analysis towards some user-perceived performance metrics, which are captured in the four Core Web Vitals metrics. 
+
+In this section we take a look at four important factors which can shed light on our understanding of how users are experiencing CMS-powered web pages in the wild:
+
+- First Contentful Paint (FCP)
+- First Input Delay (FID)
+- Cumulative Layout Shift (CLS)
+- Lighthouse scores, which cover Progressive Web Apps, and Accessibility
+
+These metrics aim to cover the core elements which are indicative of a great web user experience. Let’s review each of these in turn.
+
+**Largest Contentful Paint**
+
+Largest Contentful Paint (LCP) measures the point when the page’s main content has likely loaded and thus the page is useful to the user. It does this by measuring the render time of the largest image or text block visible within the viewport.
+
+This is different to First Contentful Pain (FCP), which measures from page load until content such as text or an image is first displayed. LCP is regarded as a good proxy for measuring when the main content of a page is loaded.
+
+A “good” LCP is regarded as under 2.5 seconds. The average website on one of the top five CMSs does not have a good LCP. Only Drupal on desktop scores over 50% here. We see major discrepancies between desktop and mobile scores: WordPress is fairly even at 33% on desktop and 25% on mobile, but Squarespace scores 37% on desktop and only 12% on mobile.
+
+IMAGE
+
+
+{{ figure_markup(
+  image="cms-real-user-largest-contentful-paint-experiences.png",
+  caption="Real-user Largest Contentful Paint experiences.",
+  description='Bar chart showing the top 5 CMSs and whether they have a "good" Largest Contentful Paint experience. Wordpress is middling with 33% on desktop and 25% on mobile, Drupal is the best with 61% on desktop and 47% on mobile, Joomla is second best with 48% on desktop and 28% on mobile, Squarespace has 37% on desktop but only 12% on mobile and Wix is the lowest with 9% on desktop and 9% on mobile.',
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTkUxfuK-FCn3_IYDJiEsKDmdmyKb0TSEhG8dFc0XBIXej1NO2uUOmd-9NCbWuh-MZ3xzMhK_kNT-4u/pubchart?oid=188727692&format=interactive",
+  sheets_gid="465267881",
+  sql_file="core_web_vitals_distribution.sql"
+  )
+}}
+
+High page weights are a possible explanation here; we’ll explore this, and the possible impact of this on the environment in a later section. It is clear here, however, that LCP performance is generally bad for CMSs, and the impact on the user experience is also bad.
```

Comment:
Had a bit of difficulty applying this feedback... to me the glass looks more than half empty! I don't think I know enough technical detail to make insightful comments here. Happy for someone else to do that if they feel better qualified.

Reply:
I think @rviscomi 's point is, rather than lamenting on the fact that the the scores are below 50% for many of the stats and being a bit of a downer can we spin this in a positive light to avoid wallowing in a bit of woe? Something like:

> Although the scores here show there is much that can be improved for LCP, this is where the huge coverage of CMSs can have a real impact for the web as a whole. Because we have shown that around 40% of websites use a CMS, if the CMS make any changes which improves LCP then suddenly a large proportion of the web will become faster as soon as those sites upgrade their CMS to include those fixes.

Or something like that. That leaves the reader with a more positive takeaway than "it's all terrible" - and maybe even nudges any CMSs reading this to take the hint! 😀

Does that make sense? Could you review your usage of "bad" and "poor" in this chapter in that light and see if there are any rewordings you want to make?

Reply:
Suggested change to spin it more positively. Feel free to alter however you see fit.
```suggestion
Even though we'd love to see CMSs performing much better here, there are still some positive takeaways from these results. For one, the fact that 61% of Drupal websites have good LCP is especially notable because it's much better than the global distribution of 48% of websites having good LCP, according to the [Chrome UX Report](https://twitter.com/ChromeUXReport/status/1293306510509039616). For 1 in 3 or 4 WordPress websites to have good LCP is also kind of amazing, given the sheer magnitude of the number of WordPress websites. Wix does have some catching up to do, but it's encouraging to see that Wix engineers are [actively](https://twitter.com/DanShappir/status/1308043752712343552) working on fixing performance issues, so this will be something to keep an eye on over the years.
```